main.py

```diff
@@ -55,15 +54,24 @@
 emptyID = config["statics"]["emojisIDs"]["empty"]
```

Comment:
https://peps.python.org/pep-0008/#constants
```suggestion
EMPTY_ID = config["statics"]["emojisIDs"]["empty"]
```